### DISCUSSION TITLE
点歌：ConvNeXt

### DISCUSSION THREAD

**Discussion by Crescent-Saturn** [mli/paper-reading#121] [General]
[A ConvNet for the 2020s](https://arxiv.org/abs/2201.03545)

> The "Roaring 20s" of visual recognition began with the introduction of Vision Transformers (ViTs), which quickly superseded ConvNets as the state-of-the-art image classification model. A vanilla ViT, on the other hand, faces difficulties when applied to general computer vision tasks such as object detection and semantic segmentation. It is the hierarchical Transformers (e.g., Swin Transformers) that reintroduced several ConvNet priors, making Transformers practically viable as a generic vision backbone and demonstrating remarkable performance on a wide variety of vision tasks. However, the effectiveness of such hybrid approaches is still largely credited to the intrinsic superiority of Transformers, rather than the inherent inductive biases of convolutions. In this work, we reexamine the design spaces and test the limits of what a pure ConvNet can achieve. We gradually "modernize" a standard ResNet toward the design of a vision Transformer, and discover several key components that contribute to the performance difference along the way. The outcome of this exploration is a family of pure ConvNet models dubbed ConvNeXt. Constructed entirely from standard ConvNet modules, ConvNeXts compete favorably with Transformers in terms of accuracy and scalability, achieving 87.8% ImageNet top-1 accuracy and outperforming Swin Transformers on COCO detection and ADE20K segmentation, while maintaining the simplicity and efficiency of standard ConvNets.

谢谢沐神！

**Comment by MedAIer** [comment]
+1  ConvNet卷土重来，上一个微信收藏还是Transformer杀入CV，今天ConvNet就卷土重来了！
> “文艺复兴”，ConvNet卷土重来，压过Transformer。本文是FAIR的Zhuang Liu(DenseNet的作者)与Saining Xie(ResNeXt的作者)关于ConvNet的最新探索，以ResNet为出发点，逐步引入近来ViT架构的一些设计理念而得到的纯ConvNet新架构ConvNeXt，取得了优于SwinT的性能，让ConvNet再次性能焕发。

谢谢沐神！

**Comment by daixiangzi** [comment]
+1
